### ISSUE TITLE
Can't find a file from archivecashes

### ISSUE THREAD

**Issue by JediKnightChan** [WolvenKit/CP77Tools#41]
Hey, can't find file base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_01\jingle.anims,12556520970865043674 nor in the whole game folder neither in basegame 4 gamedata archive. What's wrong with it and what is the hash algorythm anyway?

**Comment by JediKnightChan** [comment]
Ok, it's actually FNV1A64

**Comment by rfuzzo** [comment]
Yeah hash also is FNV1a64, I’ll add the file to the hash list. Thanks you

**Comment by JediKnightChan** [comment]
Sorry? I meant that this filepath is already in your hash list and it's even specified in some binary game files, but there is no such name hash in the game archives. (Your program seems not to throw warnings when hash is missing, but it doesn't extract anything if so, and I also checked with my own python script)

**Comment by rfuzzo** [comment]
I misunderstood. You're right, that file is not in any archive. The only explanation I can think of is that the file was removed in one of the patches.

**Comment by JediKnightChan** [comment]
That would be strange because I have the first game version and it still doesn't contain the file. 

While trying to extract the in-game movie, I found out that ".anims" files contain sound files paths, so there should be an ".anims" file for this "jingle", they can't just remove it so the movie would still work.

**Comment by Ekey** [comment]
This resource contained in **lang_XXXXX_voice.archive archives**, where XXXXX is a language ;)

**Comment by rfuzzo** [comment]
hmm. that is strange... I honestly don't know why such a specific filename incorrectly made it's way into the hashlist.

**Comment by Ekey** [comment]
@rfuzzo this filename is valid, just he is looking for it in the wrong archive :)

**Comment by rfuzzo** [comment]
hmm still in lang_en_voice.archive, there is only a nested path `lang_en_voice\base\localization\en-us\lipsync\base\media\fluff\scenes\safe_n_sound` like so. but no jingle file in there either

**Comment by Ekey** [comment]
hmmm. In my generic list, it also exists in several paths like:

```
base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_01\jingle.anims
base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_02\jingle.anims
base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_03\jingle.anims
base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_04\jingle.anims
base\media\fluff\scenes\safe_n_sound\lipsync\en\safe_n_sound_05\jingle.anims
```

but I did not find these files in any archive

**Comment by rfuzzo** [comment]
I think the gam e must read them from the relative, nested path. So, in theory, if we add `lang_en_voice\base\localization\en-us\lipsync\` in front of it, it might work?

**Comment by HitmanHimself** [comment]
i am using v0.1.4.1, this file: base\characters\main_npc\judy\h0_001_wa_c__judy\h0_001_wa_c__judy_skeleton.rig hash~169153644637331241 is not extracted from the archive, even tho this file is mentioned in the archivehashes.txt, tried both ways by extracting the full archive and by using the single hash method like this: `archive -e --hash 169153644637331241 -p "E:\downloads\basegame_4_gamedata.archive"` but the file is not extracted

**Comment by JediKnightChan** [comment]
Even if we don't know why exactly such files, mentioned in another files, don't exist, devs should run a full check for non-existing files in hashlist to exclude them.

**Comment by rfuzzo** [comment]
Yeah that’s probably a good idea.
